assets/src/edit-story/components/library/panes/media/common/attribution.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+import { useCallback, useState } from 'react';
+
+const Container = styled.a`
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: calc(100%);
```

Comment:
Can't this just be `width: 100%`?

Reply:
Done. Somehow in a different setup it was not working without it.